pandas/core/arrays/arrow/accessors.py

```diff
@@ -46,7 +46,7 @@ def _is_valid_pyarrow_dtype(self, pyarrow_dtype) -> bool:
 
     def _validate(self, data) -> None:
         dtype = data.dtype
-        if not isinstance(dtype, ArrowDtype):
```

Comment:
I'm only unsure about this one; `ArrowDtype` is conditionally imported in this file, but I don't know if this should be `if pa_version_under10p1 or not isinstance9dtype, ArrowDtype):`?